### ISSUE TITLE
Bug: Adding an IMAP account during "onboarding" hangs the app.

### ISSUE THREAD

**Issue by tetious** [Foundry376/Mailspring#1838]
**Describe the bug**
Adding an IMAP account during "onboarding" hangs the app.

**To Reproduce**
1. Start the app.
2. Add IMAP mail account. (Fastmail)
3. Fix incorrect username after initial add fails. (email address != username in my case)
4. Very quick success message, followed by a white box and the following error on console:

```
{ TypeError: Cannot read property 'emailAddress' of undefined
    at _AccountStore.addAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/src/flux/stores/account-store.ts:223:18)
    at OnboardingStore._onFinishAndAddAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/onboarding-store.ts:129:20)
    at EventEmitter.eventHandler (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:36:22)
    at EventEmitter.emit (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/eventemitter3/index.js:72:35)
    at Function.trigger (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:52:26)
    at Object.functor [as finishAndAddAccount] (/snap/mailspring/440/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/createAction.js:32:56)
    at onboarding_helpers_1.finalizeAndValidateAccount.then.validated (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/decorators/create-page-for-form.tsx:170:29)
  message: 'Cannot read property \'emailAddress\' of undefined' } { pluginIds: [ 'onboarding' ] }
```

Results in a whitebox and total hang of the app.

**Expected behavior**
The account is added.

**Setup: (please complete the following information):**
 - OS: Manjaro
 - Mailspring Version: 1.7.4 (latest snap)

**Comment by pskopnik** [comment]
#1817

**Comment by pankus** [comment]
#112

**Comment by s98330q** [comment]
Same thing occurs with Fastmail on LMDE3.  also using latest snap.  Happens with the .deb package too.

**Comment by dale76uk** [comment]
same here, fresh installation of Ubuntu 18.04.  Cannot setup new account.  Have cancelled 'Pro' version until this is resolved as I'm not able to access my email.

**Comment by oDevArc** [comment]
use the latest edge version which solve this issue : 
`sudo snap install mailspring --edge`

**Comment by bengotow** [comment]
Hey folks! Thanks for reporting this - I think it's related to a bug in the current release that causes issues with IMAP account auth. I'm really sorry it's taken so long for the fix in #1876 to get shipped. I will push out a 1.7.5 release tomorrow.

I think we can close it as a dup of #1848 but I'll keep it open until 1.7.5 ships for here for visibility. Thanks everyone for the comments and help triaging!

**Comment by bengotow** [comment]
Hey folks, thanks for reporting this. Happy to say this has been resolved in 1.7.6 which shipped this afternoon. 🚀 The app should auto-update, but if it doesn't you can also grab the new release from GitHub's Releases tab or from https://getmailspring.com/download. Thanks for your patience and help tracking this down, enjoy the update!
